solr/CHANGES.txt

```diff
@@ -89,6 +89,8 @@ Improvements
 
 * SOLR-16361: mod() is now accurate for all integers, floats, doubles and longs upto 2^52 (Dan Rosher via Eric Pugh)
 
+* SOLR-16404: Prometheus Exporter: Use non-deprecated Solr clients. (Houston Putman)
```

Comment:
From the user's point of view, you've switched to HTTP2; right?  That's more interesting to readers than what's deprecated and not; otherwise it's a refactoring (no real impact).

Reply:
Fixed